Client/src/Common/Properties.cs

```diff
@@ -88,7 +88,7 @@ public class Properties
                     string      clientKeyPem   = null,
                     string      clientP12      = null,
                     string      caCertPem      = null,
-                    bool        sslValidation  = true)
+                    bool?       sslValidation  = null)
```

Comment:
why nullable ?

Reply:
true or false by default is enough

Reply:
Because not overriding the value is not the same as overriding with true or false.
Either we don't override, or we override with true or false